pkg/grpc/tracing/tracing.go

```diff
@@ -467,9 +466,9 @@ func (msg *MsgGenericTracepointUnix) Retry(internal *process.ProcessInternal, ev
 
 func familyString(family uint16) string {
 	switch family {
-	case unix.AF_INET:
+	case constants.AF_INET:
 		return "AF_INET"
-	case unix.AF_INET6:
+	case constants.AF_INET6:
 		return "AF_INET6"
```

Comment:
could you please explain what's the logic here? is there unix.AF_INET equivalent in windows? 

it seems like linux specific parts should go to x_linux.go instead of adding constants package, but I think I dont understand the reason for constant package yet